DxDispatch/src/dxdispatch/DmlDispatchable.cpp

```diff
@@ -81,18 +111,29 @@ void FillBindingData(
             for (auto& source : sources)
             {
                 assert(source.resource != nullptr);
-                assert(source.resourceDesc != nullptr);
+                
+               if (isSerializedGraph)
 
-                if (!std::holds_alternative<Model::BufferDesc>(source.resourceDesc->value))
                 {
-                    throw std::invalid_argument("DML operators only support buffer bindings");
+                    bindingData.bufferBindings[bufferIndex].Buffer = source.resource;
+                    bindingData.bufferBindings[bufferIndex].Offset = source.elementOffset * source.elementSizeInBytes;
+                    bindingData.bufferBindings[bufferIndex].SizeInBytes = source.elementCount * source.elementSizeInBytes;
                 }
+                else
+                {
+                    assert(source.resourceDesc != nullptr);
+                    if (!std::holds_alternative<Model::BufferDesc>(source.resourceDesc->value))
+                    {
+                        throw std::invalid_argument("DML operators only support buffer bindings");
+                    }
 
-                auto& bufferDesc = std::get<Model::BufferDesc>(source.resourceDesc->value);
+                    auto& bufferDesc = std::get<Model::BufferDesc>(source.resourceDesc->value);
+
+                    bindingData.bufferBindings[bufferIndex].Buffer = source.resource;
+                    bindingData.bufferBindings[bufferIndex].Offset = source.elementOffset * source.elementSizeInBytes;
+                    bindingData.bufferBindings[bufferIndex].SizeInBytes = bufferDesc.sizeInBytes - bindingData.bufferBindings[bufferIndex].Offset;
```

Comment:
Can you please verify whether for DmlDispatchable also we can use the same calculation for `SizeInBytes` as DmlSerializedGraphDispatchble? If yes, then we can get rid of the if/else.